### ISSUE TITLE
Missing CustomMongoObserver

### ISSUE THREAD

**Issue by TomVeniat** [emited/tmd_framework#3]
Hi,
It seems that there is a class that should be in the project but  I can't find it :
https://github.com/emited/tmd_framework/blob/11b1e05e8efc21fd2cc33a55585856b301659648/src/utils/external_resources.py#L7
We should either add the class or modify the import path.

**Comment by emited** [comment]
Yes, this is a bug. CustomMongoObserver was made in order to not capture stdout. But this is possible by setting sacred's option to 'no' by writing '--capture no' in the commandline.

**Comment by TomVeniat** [comment]
Thanks for the reply, I'll remove it then.
btw you can also avoid capturing the output using `SACRED_SETTINGS`:
``` python
from sacred import SETTINGS as SACRED_SETTINGS
SACRED_SETTINGS['CAPTURE_MODE'] = 'no'
```

**Comment by TomVeniat** [comment]
Fixed by 08cd389
